build.sc

```diff
@@ -38,7 +39,11 @@ val scalaNativeVersions = Seq(
 )
 
 trait CommonModule extends ScalaModule {
-  def scalacOptions = T{ if (scalaVersion() == scala212) Seq("-opt:l:method") else Nil }
+  def scalacOptions = T{
+    super.scalacOptions() ++ {
```

Comment:
To avoid loosing the `-scalajs` option, I added  `super.scalacOptions() ++` everywhere.